### ISSUE TITLE
The "scope" names "variable.other.member" includes the dot separator

### ISSUE THREAD

**Issue by Jaykul** [PowerShell/EditorSyntax#131]
First off: I think we should be lexing a `property` and a `method` rather than the generic `member` -- however, this bug is simply against the fact that the **separator** dot is being included in the scope.

I'm using VS Code 1.26.0-insider

For repro sake, I added this to my preference:

```json
    "editor.tokenColorCustomizations": {
        "textMateRules": [
            {
                "scope": "variable.other.member",
                "settings": {
                    "foreground": "#e23de2"
                }
            },
        ]
    },
```

And got this:

![image](https://user-images.githubusercontent.com/192942/44064092-60bcae88-9f31-11e8-8742-5b008788407e.png)

I expected only the words to be pink, i.e. only "Length" not ".Length" in:
```posh
$file.Length
```

I also did not expect it to color the method `Delete()`

**Comment by omniomi** [comment]
Good catch. The separator actually has its own scope that it should use (`punctuation.separator...`,) I must have missed that.

The property vs method question is interesting and `variable.function` does exist for methods.

https://www.sublimetext.com/docs/3/scope_naming.html#variable

**Comment by keith-hall** [comment]
for reference, the scope of the `.` was also mentioned at https://github.com/PowerShell/EditorSyntax/issues/107#issuecomment-393640603
